docs/content/faq.md

```diff
@@ -58,3 +58,38 @@ and make sure that `protocolUseSSL` / `CMD_PROTOCOL_USESSL` is set to `true` if
 ## HedgeDoc fails executing migrations and does not start
 Unfortunately, older versions of HedgeDoc had some bugs regarding migrations and didn't always record that a migration was executed.  
 Have a look at the *[Troubleshooting Migrations](/guides/migration-troubleshooting/)* guide for more information.
+
+## Why does my interface look weird?
+
+If your interface looks like this
+
+![Broken HedgeDoc interface](./faq/interface_broken.png)
+
+instead of this
+
+![Not broken HedgeDoc interface](./faq/interface_okay.png)
+
+then HedgeDoc can't find the asset files like images, stylesheets, fonts, etc.
+You should check the following settings (or their corresponding environment variables if you e.g. use docker.):
+
+- `domain` (env: `CMD_DOMAIN`)
+- `protocolUseSSL` (env: `CMD_PROTOCOL_USESSL`)
+- `urlAddPort` (env: `CMD_URL_ADDPORT`)
+
+Check [the configuration page](/configuration/) for an explanation.
+
+These settings control how HedgeDoc generates links to other pages and to assets, therefore you must set them from the outside point of view.
+
+Example: You access your HedgeDoc instance directly via `http://192.0.2.1:3000`. You must set:
```

Comment:
Does our hugo theme support alert blocks like HedgeDoc does? Maybe these examples would be a good canidate for this.

Reply:
It does afaik but I don't know if that's necessary here.

Reply:
I added example blocks